pkg/apis/broker/register.go

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
It would work with the vendored broker APIs, but having our own types here allows adding additional status fields like topic id and subscription id. I think this will make debugging easier for the user, though it's a nice-to-have not a requirement.

Currently these custom fields don't work without some changes to the eventing webhook.

Reply:
Having additional status seems good to me. I am not sure compatibility to OSS eventing APIs is a requirement or not. I guess we should be careful not to make any incompatible change?